README.md

```diff
@@ -7,28 +7,35 @@ Socket.IO-client for iOS/OS X.
 ```swift
 import SocketIO
 
-let manager = SocketManager(socketURL: URL(string: "http://localhost:8080")!, config: [.log(true), .compress])
-let socket = manager.defaultSocket
+class SocketIOManager{
+    
+    let manager = SocketManager(socketURL: URL(string: "http://localhost:8080")!, config: [.log(true), .compress])
```

Comment:
it's better to safely unwrap stuff like URL's, you don't have to, but I'd suggest it :D